common/src/main/scala/uk/ac/wellcome/platform/archive/common/storage/services/StorageManifestService.scala

```diff
@@ -23,22 +24,23 @@ import uk.ac.wellcome.platform.archive.common.storage.models.{
   FileManifest,
   StorageManifest
 }
+import uk.ac.wellcome.storage.ObjectLocation
 
 import scala.concurrent.{ExecutionContext, Future}
 
 class StorageManifestService(
   implicit
   executionContext: ExecutionContext,
   s3Client: AmazonS3
-) {
+) extends Logging {
```

Comment:
Why did you add Logging? I can’t see where it’s used.

Reply:
oh yes, think I was debugging.